src/analyzer/analyzersilence.cpp

```diff
@@ -10,33 +10,58 @@ namespace {
 // verify that the track samples have not changed since the last analysis
 constexpr CSAMPLE kSilenceThreshold = 0.001f; // -60 dB
 // TODO: Change the above line to:
-//constexpr CSAMPLE kSilenceThreshold = db2ratio(-60.0f);
+// constexpr CSAMPLE kSilenceThreshold = db2ratio(-60.0f);
+
+// These are in dBV expressed as Volts RMS (which seems, sensibly,
+// the way Mixxx works).
+// Don't change these to const as they are used only to feed the
+// fade thresholds which are consts below themselves while we work
+// out which one is best, and you'll just be adding to exe size.
```

Comment:
The whole thing is a comment in case the chosen values are not the best.  I should just delete the lot. I believe I have good values here.

Reply:
> I am a bit puzzled how this iteracts with manual set intros and outros.

You asked about this before.  It does not interact or use manual set intros and outros cues at all.  It runs as discussed in [ AutoDj: New transition mode 'Radio Laneway' #13716](https://github.com/mixxxdj/mixxx/issues/13716) 

I've yet to find a track where it isn't 'good enough', and in some cases does consistently better than me (where I might miss 1 in 10 cold endings), so I'm not sure I agree with you about not working with ambient noise at the beginning or end, but since I can't claim any great familiarity with club music I will leave that for you to decide.  

Why not try it and see?